src/tests/ftest/deployment/agent_failure.yaml

```
@@ -0,0 +1,70 @@
+hosts:
+    test_servers:
+        - server-A
+        - server-B
+        - server-C
+        - server-D
+        - server-E
+        - server-F
+        - server-G
+    test_clients:
+        - client-A
+
+timeout: 300
+
+setup:
+    start_agents_once: False
+
+daos_server:
+  pattern_timeout: 60
+
+server_config:
+    name: daos_server
+    engines_per_host: 2
+    servers:
+        0:
+            pinned_numa_node: 0
+            nr_xs_helpers: 1
+            fabric_iface: ib0
+            fabric_iface_port: 31317
+            log_file: daos_server0.log
+            bdev_class: nvme
+            bdev_list: ["0000:00:00.0"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem0"]
+            scm_mount: /mnt/daos0
+        1:
+            pinned_numa_node: 1
+            nr_xs_helpers: 1
+            fabric_iface: ib1
+            fabric_iface_port: 31417
+            log_file: daos_server1.log
+            bdev_class: nvme
+            bdev_list: ["0000:00:00.1"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem1"]
+            scm_mount: /mnt/daos1
+
+pool:
+    size: 100%
+    control_method: dmg
+
+container:
+    type: POSIX
+    control_method: daos
+
+ior:
+    client_processes:
+        np: 4
```

Could we bump the processes? E.g. 32 client PPN would provide better coverage (more race conditions, etc.)
```suggestion
        ppn: 32
```
Also, it seems the test didn't pick this up correctly:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8368/10/artifact/Functional%20Hardware%20Large/deployment/agent_failure.py/job.log
`-np 1 ior`

---

I'll use `ppn` instead. Thanks for checking the log.